src/components/floating-menu/floating-menu.js

```diff
@@ -169,6 +183,7 @@ class FloatingMenu extends mixin(createComponent, eventedShowHideState) {
       }
       this._getContainer().appendChild(this.element);
       this._place();
+      (this.element.querySelector(this.options.selectorPrimaryFocus) || this.element).focus();
```

Comment:
If `selectorPrimaryFocus` is unspecified then focus should be placed on the first focusable element inside the menu.